frontend/tests/jest/tests/ProductSection.test.tsx

```diff
@@ -118,6 +118,33 @@ describe('ProductSection Tests', () => {
          (expect(note) as any).not.toBeInTheDocument();
          (expect(disclaimer) as any).not.toBeInTheDocument();
       });
+
+      test('compatibility renders', async () => {
+         renderWithAppContext(
+            <ProductSection
+               product={getMockProduct()}
+               selectedVariant={getMockProductVariant()}
+               onVariantChange={jest.fn()}
+               internationalBuyBox={null}
+            />
+         );
+
+         expect(
+            screen.getByTestId('product-compatibility-dropdown')
+         ).not.toBeVisible();
+
+         const compatibilityButton = await screen.findByTestId(
+            'product-compatibility-dropdown-button'
+         );
+
+         await act(() => compatibilityButton.click());
```

Comment:
I am curious why you are using `act` instead of chaining the command.

```
await screen.findByTestId(
            'product-compatibility-dropdown-button'
         ).click()
```

Reply:
Seems to work just fine like that. I'll make the change. Thanks!